### ISSUE TITLE
Cannot set the PROJECTION value correctly

### ISSUE THREAD

**Issue by janpisl** [geographika/mappyfile#63]
This is (a part of) code I use to open a template of a MapFile and set some values:

```
import mappyfile

d = mappyfile.open('test.map')


d["NAME"] = "testpg1"
d["PROJECTION"] = "init=epsg:4326" 

```
This is (a part of) the resulting MapFile:
```
MAP
    NAME "testpg1"
    PROJECTION
        "i"
        "n"
        "i"
        "t"
        "="
        "e"
        "p"
        "s"
        "g"
        ":"
        "4"
        "3"
        "2"
        "6"
    END
```

For some reason, the value I set for PROJECTION appears in the MapFile like this - each character on a new line. All the other parameters I set (like NAME) are ok. I have tried with several different MapFiles and it behaved the same way. 

I am new to MapServer and mappyfile so there might be something very obvious that I am overlooking.

**Comment by geographika** [comment]
Hi,

The following should work:
`
d["PROJECTION"] = ["init=epsg:4326"]
`

There is an issue in that both lists and strings should be allowed, so I will need to fix the pretty printer to allow this - thanks for reporting it. 

The list option is to allow projection blocks such as:
```

  PROJECTION
    "proj=utm"
    "zone=12"
    "ellps=WGS84"
    "towgs84=1,0,0"
  END

```

**Comment by geographika** [comment]
Fixed in 0.7.2 release
